src/main/resources/db/migration/V003__CREATE_BOOK.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE book
     date_started_reading  DATE,
     edition               VARCHAR(255),
     book_genre            INTEGER,
+    book_format           INTEGER,
```

Comment:
Instead of modifying this migration, could you please create a new migration for this (for an example, see v14 on master)?

Reply:
Alright will review all this